src/quickgui/qgsquickpositionkit.h

```diff
@@ -159,6 +164,9 @@ class QUICK_EXPORT QgsQuickPositionKit : public QObject
     //! \copydoc QgsQuickPositionKit::simulatePositionLongLatRad
     void setSimulatePositionLongLatRad( const QVector<double> &simulatePositionLongLatRad );
 
+    //! \copydoc QgsQuickPositionKit::source
```

Comment:
The \copydoc above is used in a strange way, I would not use it as inspiration :-)

How about comment like "Returns pointer to the internal QGeoPositionInfoSource object used to receive GPS location."

Reply:
I think it's aiming towards "documenting properties" instead of "documenting the logic".

We've got way too many "returns pointer to the internal receiver", while it would be much more interesting to know what "receiver" is used for (see e.g. like you already used too different descriptions in the two API proposals in the comments in here ;) ).

But yes, `\copydoc` unortunately is neither supported enough for this kind of usage of directly on the property by doxygen nor is Qt Creator smart enough to make them accessible while working. So I normally copy paste a simple "this property gives access to ..." among getters / setters / notification signal.

This is more a general thought than a request for any additional change here :)

Reply:
Actually, one thing that would be very valuable for an API doc is about the lifetime and ownership of the pointer and that it may only be used until the sourceChanged signal has been emitted.